From 39a124f8145c8751af95d4017b292d6b4ef423a1 Mon Sep 17 00:00:00 2001
From: Jouni Malinen <quic_jouni@quicinc.com>
Date: Tue, 1 Feb 2022 19:58:40 +0200
Subject: [PATCH 5/5] EAP-pwd: Fix the prefix in a debug message

This was copied from sae.c, but the debug message prefix was not changed
to match the use here.

Signed-off-by: Jouni Malinen <quic_jouni@quicinc.com>
Signed-off-by: Rameshkumar Sundaram <quic_ramess@quicinc.com>
(cherry picked from commit 34575ad72ebbb5411e394af7090f79a27a60a9aa)
---
 src/eap_common/eap_pwd_common.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/eap_common/eap_pwd_common.c b/src/eap_common/eap_pwd_common.c
index ff22b29..0e3a7b2 100644
--- a/src/eap_common/eap_pwd_common.c
+++ b/src/eap_common/eap_pwd_common.c
@@ -275,7 +275,7 @@ int compute_password_element(EAP_PWD_group *grp, u16 num,
 	    crypto_bignum_sub(prime, y, y) < 0 ||
 	    crypto_bignum_to_bin(y, x_y + MAX_ECC_PRIME_LEN,
 				 MAX_ECC_PRIME_LEN, primebytelen) < 0) {
-		wpa_printf(MSG_DEBUG, "SAE: Could not solve y");
+		wpa_printf(MSG_DEBUG, "EAP-pwd: Could not solve y");
 		goto fail;
 	}
 
-- 
2.7.4

